samples/security/spire/README.md

```diff
@@ -0,0 +1,44 @@
+# Integrating SPIRE through Envoy SDS
+
+There are currently multiple forms of integrating third-party Certificate Authorities (CA) whitin Istio. This example deploys a setup of [SPIRE](https://github.com/spiffe/spire) (the SPIFFE Runtime Environment) as an Exernal CA, through [Envoy's SDS API integration](https://github.com/alexandrealvino/istio.io/blob/6eff1cd45dfce244978cafc10a88f7620c0438fc/content/en/docs/ops/integrations/spire/index.md).
+
+To assist this deployment, two additional components will be shipped together with SPIRE, the [CSI Driver](https://github.com/spiffe/spiffe-csi) and the [Kubernetes Workload Registrar](https://github.com/spiffe/spire/tree/main/support/k8s/k8s-workload-registrar).
+
+The desired behaviour will get triggered during istio-agent bootstrap, in case a socket file under `/var/run/secrets/workload-spiffe-uds/socket` is detected. This tells istio-agent **not** to start its own SDS server, but to use the one provided via socket.
+
+
+## Usage
+
```

Comment:
Regarding the first 4 bullets: 

I was originally very tempted to put something like this, but after taking a look at the similar docs under `/samples` there are no appearances of Kubernetes as pre requisite. I believe it goes without saying since istio is natively for k8s. 

> What repo to download to get the spire.yaml and profile.yaml files.

They will live on the same directory of this README.

Reply:
You're right, most of other READMEs in the /samples directory don't include detailed prerequisite info. I did see that [httpbin](https://github.com/istio/istio/tree/master/samples/httpbin) and [tcp-echo](https://github.com/istio/istio/tree/master/samples/tcp-echo) at least say "Install Istio by following the istio install instructions." Also, [kind-lb](https://github.com/istio/istio/tree/master/samples/kind-lb) has a short dependency section.  Maybe we could at least say Istio must be installed. It could be the first step if you want to save space instead of having a Prerequisites section.

I agree the repo info is unnecessary because GitHub will show those files at the top of the page because they're in the same directory. 

Regarding the shell type, I was concerned that some other shells might not support `export` but I see at least zsh does.
 
Most of the other bullets I listed don't seem to be covered in the other /samples READMEs. In my world, they would be. 😄 Looks like the other Mithril doc does have a prerequisites section, so that's good.

Reply:
> Maybe we could at least say Istio must be installed. It could be the first step if you want to save space instead of having a Prerequisites section.

Regarding this concern, what I think is the most important info here is that SPIRE needs to be up and running prior to installing Istio, which is probably something not as usual for the user and may cause confusion. Do you think that this is clear enough at this point?